src/features/channels/components/channel-list/about/about.tsx

```diff
@@ -1,20 +1,42 @@
+import useOnMount from '@metis/common/hooks/useOnMount';
+import httpService from '@metis/common/services/http.service';
+import Grid from '@mui/material/Grid';
 import Dialog from '@mui/material/Dialog';
 import DialogContent from '@mui/material/DialogContent';
 import DialogContentText from '@mui/material/DialogContentText';
 import DialogTitle from '@mui/material/DialogTitle';
+import Divider from '@mui/material/Divider';
 import Typography from '@mui/material/Typography';
-import * as React from 'react';
+import { useState, useEffect } from 'react';
 import useStyles from './about.styles';
 
+export interface WebAppInfo {
+  name: string;
+  version: string;
+}
+
 type Props = {
   title: string;
   message: string;
   onClick?: () => void;
 };
 
+const loadWebAppInfo = async (): Promise<WebAppInfo[]> => {
+  const response = await httpService.get<WebAppInfo[]>('/v1/api/version');
+  return response.data;
+};
+
 const About = ({ title, message, onClick }: Props) => {
-  const [open, setOpen] = React.useState(false);
+  const [user, setUser] = useState<WebAppInfo[]>();
```

Comment:
Why is this state called user? We're setting this to the application's info, not users. Try to use a better name for the state so the code is easier to understand

Reply:
That's correct I will fix it.